pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1080,9 +1082,13 @@ func (og *operationGenerator) GenerateMapVolumeFunc(
 			return volumeToMount.GenerateError("MapVolume type assertion error", fmt.Errorf("volume host does not implement KubeletVolumeHost interface"))
 		}
 		hu := kvh.GetHostUtil()
-		devicePath, err = hu.EvalHostSymlinks(devicePath)
+		resolvedDevicePath, err := hu.EvalHostSymlinks(devicePath)
 		if err != nil {
-			return volumeToMount.GenerateError("MapVolume.EvalHostSymlinks failed", err)
+			// For CSI plugin, device won't be available at devicePath until MapPodDevice is called,
+			// so, just warn and continue.
+			klog.Warningf("MapVolume.EvalHostSymlinks failed for %s: %v", devicePath, err)
```

Comment:
Please lower the log level, this will be printed for every CSI block volume setup.